README.md

```diff
@@ -25,42 +24,129 @@ In the browser, the promise handler for
 ```
 
 How you send the PushSubscription data to your backend, store it
 referenced to the user who requested it, and recall it when there's
-new a new push subscription update is left as an excerise for the
+new a new push subscription update is left as an exercise for the
 reader.
 
-The data can be any serial content (string, bit array, serialized
-JSON, etc), but be sure that your receiving application is able to
-parse and understand it. (e.g. `data = "Mary had a little lamb."`)
+### Sending Data using `webpush()` One Call
 
-gcm_key is the API key obtained from the Google Developer Console.
-It is only needed if endpoint is
-https://android.googleapis.com/gcm/send
+In many cases, your code will be sending a single message to many
+recipients. There's a "One Call" function which will make things
+easier.
 
-`headers` is a `dict`ionary of additional HTTP header values (e.g.
-[VAPID](https://github.com/mozilla-services/vapid/tree/master/python)
-self identification headers). It is optional and may be omitted.
+```pythonstub
+    from pywebpush import webpush
 
-to send:
+    webpush(subscription_info,
+            data,
+            vapid_private_key="Private Key or File Path[1]",
+            vapid_claims={"sub": "mailto:YourEmailAddress"})
 ```
-WebPusher(subscription_info).send(data, headers)
+This will encode `data`, add the appropriate VAPID auth headers if required and send it to the push server identified
+in the `subscription_info` block.
+
+**Parameters**
+
+*subscription_info* - The `dict` of the subscription info (described above).
+
+*data* - can be any serial content (string, bit array, serialized JSON, etc), but be sure that your receiving
+application is able to parse and understand it. (e.g. `data = "Mary had a little lamb."`)
+
+*vapid_claims* - a `dict` containing the VAPID claims required for authorization (See
+[py_vapid](https://github.com/web-push-libs/vapid/tree/master/python) for more details)
+
+*vapid_private_key* - Either a path to a VAPID EC2 private key PEM file, or a string containing the DER representation.
+(See [py_vapid](https://github.com/web-push-libs/vapid/tree/master/python) for more details.) The `private_key` may be
+a base64 encoded DER formatted private key, or the path to an OpenSSL exported private key file.
+
+e.g. the output of:
+```commandline
+openssl ecparam -name prime256v1 -genkey -noout -out private_key.pem
 ```
-to send for Chrome:
+
+**Example**
+
+```pythonstub
+from pywebpush import webpush, WebPushException
+
+try:
+    webpush(
+        subscription_info={
+            "endpoint": "https://push.example.com/v1/12345",
+            "keys": {
+                "p256dh": "0123abcde...",
+                "auth": "abc123..."
+            }},
+        data="Mary had a little lamb, with a nice mint jelly",
+        vapid_private_key="path/to/vapid_private.pem",
+        vapid_claims={
+                "sub": "YourNameHere@example.org",
+                "aud": "https://push.example.com/"
```

Comment:
Are you expecting the user to define this? OR does the library figure this out on their behalf?

If the latter, could this be removed as it's not necessary, likely to cause confusion and is different from the previous example.